docs/articles/recommender_from_clickstreams.md

```diff
@@ -0,0 +1,226 @@
+The idea is simple: 
+>If you only have *positive* rows consisting of neighbors, just make up some random pairs of hotels that were not seen next to each other in a sequence.
+This can look like this:
+
+| Focus                   | Other Hotel             | Neighbor? |
+|-------------------------|-------------------------|-----------|
+| Tranquil Haven Resort   | Summit Vista Inn        | Yes       |
+| Tranquil Haven Resort   | Some Other Hotel 1      | No        |
+| Summit Vista Inn        | Tranquil Haven Resort   | Yes       |
+| Summit Vista Inn        | Some Other Hotel 2      | No        |
+| Summit Vista Inn        | Azure Sands Hotel       | Yes       |
+| Summit Vista Inn        | Some Other Hotel 1      | No        |
+| Azure Sands Hotel       | Summit Vista Inn        | Yes       |
+| Azure Sands Hotel       | Some Other Hotel 3      | No        |
+| Azure Sands Hotel       | Evergreen Retreat Lodge | Yes       |
+| Azure Sands Hotel       | Some Other Hotel 3      | No        |
+| Evergreen Retreat Lodge | Azure Sands Hotel       | Yes       |
+| Evergreen Retreat Lodge | Some Other Hotel 1      | No        |
+
+And now, we can build a binary classifier that has to solve the reasonable classification task of finding out whether two hotels were clicked successively, or not.
+
+And this is what happens under the hood in a nutshell. Of course, there are many more hyperparameters that you can tune to get better embeddings. We have seen the embedding size (called `vector_size` in Gensim) already. Another important hyperparameter is **window size** using the `window_size` parameter. This enlarges the focus of the model, so we are not only interested in the direct left and right neighbors of the hotel but also hotels that are maybe two or three steps apart from the focus element.
+
+## Conclusion
+We have seen how easy it is to get embeddings for hotels - or rather any type of item - that users click on in an online session. We only have to record the clickstream data and preprocess it a bit, so we can use Gensim to train a model in a few lines of code.
+
+It is probably one of the easiest projects to set up, and still, you get really interesting results from it in the form of embeddings. You can then use these embeddings to either build a recommender system, but you can also just visualize them to gain more insights into your data.
+
+As an example, you can use PCA, t-SNE, or UMAP to project the embeddings into two dimensions like this:
+```python
+import matplotlib.pyplot as plt
+from umap import UMAP
+
+t = UMAP(random_state=0).fit(model.wv.vectors)
+plt.scatter(t.embedding_[:, 0], t.embedding_[:, 1])
+```
+
+![Projected Embeddings](../assets/use_cases/recommender_from_clickstreams/scatter.png)
+
+Here, you can see how certain hotels form clusters, and it might be interesting to study them. But we will stop at this point for now. Give it a try with your datasets now! All it takes are about ten lines of code. Thanks for reading!
+
```

Comment:
Do you want to try interpreting this?

Reply:
Sure, I can elaborate on that a bit. I would write something like "depending on the data, close hotels could be interpreted as alternatives".

Reply:
Will do a quick commit now. 
Can you indicate here in this convo what exactly you would add at line 232 ?

Reply:
"Here, you can see how certain hotels form clusters. In general, we cannot say what they mean because it boils down to what it means if two hotel embeddings are close to each other. You have to investigate the data further to come to an interpretation of these clusters.

But as a concrete example, imagine that the customers usually check for hotel alternatives, i.e., they want to stay in a city, and check many hotels that might be interesting to them, but that have quite similar characteristics, for example the price range, or having a private bathroom. In this case, embeddings of hotels that the customers think have similar characteristics are close to each other, and we could use the embeddings to build a recommended for alternative hotels.

Give it a try with your datasets now! All it takes are about ten lines of code. Thanks for reading!"